Utilities/O2device/include/O2device/O2device.h

```diff
@@ -0,0 +1,67 @@
+/********************************************************************************
+ *    Copyright (C) 2014 Frankfurt Institute for Advanced Studies               *
+ *                                                                              *
+ *              This software is distributed under the terms of the             *
+ *         GNU Lesser General Public Licence version 3 (LGPL) version 3,        *
+ *                  copied verbatim in the file "LICENSE"                       *
+ ********************************************************************************/
+/**
+ * O2device.h
+ *
+ * @since 2014-12-10
+ * @author M. Krzewicki <mkrzewic@cern.ch>
+ */
+
+#ifndef O2DEVICE_H_
```

Comment:
I have brought up the idea of a common O2 device base class with specific O2 functionality before, but not all did really like it, or at least see the necessity. So I hope more people can comment on this. I support the idea, even if it introduces another layer.